app/serializers/v1/predefined_participant_serializer.rb

```diff
@@ -11,6 +11,14 @@ class V1::PredefinedParticipantSerializer < V1Serializer
     object.predefined_user_parameter.slug
   end
 
+  attribute :auto_invitation do |object|
```

Comment:
I wonder can we wrap it into an `%i[slug auto_inviation ...].each |attribute|` block and use `object.predefined_user_paramter.send(attribute)` 🤔